README.md

```diff
@@ -0,0 +1,46 @@
+# Neurobagel subcommunities
+
+[![GitHub Actions Workflow Status](https://img.shields.io/github/actions/workflow/status/neurobagel/communities/update_enigma_assessments.yaml?style=flat-square&label=ENIGMA-PD&link=https%3A%2F%2Fgithub.com%2Fneurobagel%2Fcommunities%2Fblob%2Fmain%2F.github%2Fworkflows%2Fupdate_enigma_assessments.yaml)](https://github.com/neurobagel/communities/blob/main/.github/workflows/update_enigma_assessments.yaml)
+
+
+Scripts and configuration files that allow Neurobagel subcommunities to define custom standardized vocabularies used among their studies or sites and make them available for use within the Neurobagel tool ecosystem.
+
+
+## To create a new subcommunity vocabulary of assessments (WIP)
+<!-- TODO: Once finalized, add instructions for creating a file with vocab namespace metadata. -->
+
+Neurobagel expects each subcommunity-curated vocabulary to be maintained in a single Google Sheets table.
+
+1. Create a new [Google Sheet](https://workspace.google.com/intl/en_ca/products/sheets/) and give it a descriptive name, e.g., `<SUBCOMMUNITY NAME> Assessment Vocabulary`. 
+
+2. Open the **Share** settings and set the access permissions for the Google Sheet to **Anyone with the link**.
+
+3. In the first worksheet (default name Sheet1), create a table for your assessment terms in the format below, with each row representing a single assessment used by your subcommunity (example assessments shown below):
```

Comment:
Should we just have our own template google sheet and ask them to make a copy and then set the share settings? Seems like that would be easier